Adds services using token to show sub-command

dposada opened the pull request:
## Changes proposed in this PR

- adding a table showing the services using the token when doing `waiter show`

## Why are we making these changes?

It makes the `show` output more useful.

## Demo

[![asciicast](https://asciinema.org/a/3pY3pojKMzHiHL7WhkFQf4NpJ.svg)](https://asciinema.org/a/3pY3pojKMzHiHL7WhkFQf4NpJ?autoplay=1)

shamsimam commented on cli/integration/tests/waiter/test_cli.py:
```diff
@@ -818,3 +819,47 @@ def test_init_existing_file(self):
+            self.assertIsNotNone(re.search(f'^{service_id_1}.+Running', cli.stdout(cp), re.MULTILINE))
+            self.assertIsNotNone(re.search(f'^{service_id_2}.+Failing', cli.stdout(cp), re.MULTILINE))
+        finally:
+            util.delete_token(self.waiter_url, token_name)
```
Does this need a `kill_services=True`?

dposada replied:
Done

dposada commented:
@shamsimam I added a `Current?` column:

```bash
$ waiter show demo

=== WAITER1 / demo ===

Last Updated: just now (dpo)

Owner                      dpo
CPUs                       0.3
Memory                     128 MiB
Version                    0.0.5
Fallback period (seconds)  300

Command:
/home/dpo/source/waiter/test-apps/kitchen/bin/kitchen -p $PORT0

# Services    2
# Failing     0
Total Memory  256 MiB
Total CPUs    0.6

Service Id                                       Run as user      CPUs  Memory    Version    Status    Last request    Current?
waiter-service-d3c1b1574500c3440533341fb86f54e0  dpo               0.3  128 MiB   0.0.5      Running   just now        ✔
waiter-service-e65bd9bde70a559c2739ebf77f236732  dpo               0.3  128 MiB   0.0.5      Starting  seconds ago     ✗
```

And I added a `--no-services` flag:

```bash
$ waiter show demo --no-services

=== WAITER1 / demo ===

Last Updated: 24 minutes ago (dpo)

Owner                      dpo
CPUs                       0.3
Memory                     128 MiB
Version                    0.0.5
Fallback period (seconds)  300

Command:
/home/dpo/source/waiter/test-apps/kitchen/bin/kitchen -p $PORT0
```
